README.md

```diff
@@ -48,11 +48,12 @@ Find out more at [wagtail.org](https://wagtail.org/).
 
 Wagtail works with [Python 3](https://www.python.org/downloads/), on any platform.
 
-To get started with using Wagtail, run the following in a virtual environment:
+To get started with using Wagtail, run the following in a [virtual environment](https://docs.python.org/3/tutorial/venv.html):
```

Comment:
This may not be exactly right, there are a few different ways to set up a virtual environment.

We may cause more confusion by being specific here. 

I'll let other core team chime in though. We do intentionally keep this brief as there main docs are the place to get full instructions. https://docs.wagtail.org/en/stable/getting_started/tutorial.html#create-and-activate-a-virtual-environment

Maybe we can write

```suggestion
To get started with using Wagtail, run the following in a virtual environment such as [venv](https://docs.python.org/3/tutorial/venv.html):
```

Reply:
I prefer @krishvsoni's original version here, and I'd point to the Diataxis framework's advice on tutorials to back this up - [Ignore options and alternatives](https://diataxis.fr/tutorials/#ignore-options-and-alternatives).

Anyone who's knowledgeable / opinionated enough to have a preferred approach to creating virtual environments will happily disregard the link and do their own thing; and for everyone else, knowing that alternatives to venv exist (but not anything about them) doesn't add any value. And, as you say, people who are not familiar with virtual environments at all are better served by the full tutorial.

Reply:
Good call, thank you.

@krishvsoni  if you are OK to clean up the other parts of the PR (just revert accidental/unrelated changes), we should be good to go on this one.